ui/components/app/asset-list/asset-list.js

```diff
@@ -175,53 +119,7 @@ const AssetList = ({ onClickAsset }) => {
             margin={4}
           />
         )}
-      {shouldShowBuy || shouldShowReceive ? (
-        <Box
-          paddingInlineStart={4}
-          paddingInlineEnd={4}
-          display={Display.Flex}
-          gap={2}
-        >
-          {
-            ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
-            shouldShowBuy ? (
-              <AssetListConversionButton
-                variant={ASSET_LIST_CONVERSION_BUTTON_VARIANT_TYPES.BUY}
-                onClick={() => {
-                  openBuyCryptoInPdapp();
-                  trackEvent({
-                    event: MetaMetricsEventName.NavBuyButtonClicked,
-                    category: MetaMetricsEventCategory.Navigation,
-                    properties: {
-                      location: 'Home',
-                      text: 'Buy',
-                      chain_id: chainId,
-                      token_symbol: defaultSwapsToken,
-                    },
-                  });
-                }}
-              />
-            ) : null
-            ///: END:ONLY_INCLUDE_IF
-          }
-          {
-            ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
-            shouldShowReceive ? (
-              <AssetListConversionButton
-                variant={ASSET_LIST_CONVERSION_BUTTON_VARIANT_TYPES.RECEIVE}
-                onClick={() => setShowReceiveModal(true)}
-              />
-            ) : null
-            ///: END:ONLY_INCLUDE_IF
-          }
-          {showReceiveModal ? (
-            <ReceiveModal
-              address={selectedAddress}
-              onClose={() => setShowReceiveModal(false)}
-            />
-          ) : null}
-        </Box>
-      ) : null}
+      {shouldShowBuy && <RampsCard variant={RAMPS_CARD_VARIANT_TYPES.TOKEN} />}
```

Comment:
Two comments here:
1. Prefer [ternary over conditional rendering](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx), as the deleted code was 
2. Previous code was fencing the ramp code to be included only for `build-main,build-beta,build-flask`, we should keep the behavior

```suggestion
      {
        ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
        shouldShowBuy ? (
          <RampsCard variant={RAMPS_CARD_VARIANT_TYPES.TOKEN} />
        ) : null
        ///: END:ONLY_INCLUDE_IF
      }
```

This would need to be paired with the import as well:
```ts
///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
import {
  RAMPS_CARD_VARIANT_TYPES,
  RampsCard,
} from '../../multichain/ramps-card/ramps-card';
///: END:ONLY_INCLUDE_IF
```

Reply:
[40eb114](https://github.com/MetaMask/metamask-extension/pull/23352/commits/40eb114da411c8685f3f6e32479c388723bc032e)